docs/java/features/resilience/resilience.md

```diff
@@ -0,0 +1,221 @@
+---
+id: resilience
+title: Resilience Capabilities of the SAP Cloud SDK for Java
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Resilience
+description: This article describes how the SAP Cloud SDK for Java can be used to introduce resilience patterns into an application.
+keywords:
+- sap
+- cloud
+- sdk
+- resilience
+- multi tenancy
+- cloud native
+- timeout
+image:
+---
+
+The SAP Cloud SDK for Java provides abstractions for some frequently used resilience patterns like timeout, retry or circuit breaker.
+Applying such patterns helps making an application more resilient against failures it might encounter.
+
+Also check out our [dedicated tutorial](https://developers.sap.com/tutorials/s4sdk-resilience.html) on the topic!
+
+## Consuming the Resilience API
```

Comment:
## Making your code resilient

Reply:
I would like to leave it like it is: Generally I don't like `making` so much. Also it's not the code that is made resilient, but it's execution.

Reply:
I don't like consuming because SAP is the first place I see it overused. Maybe somewhere in EPAM and Oracle, it's also like this. I'm also not a Java guy, where it might be a key term for using any API. It just makes it all very generic. Again, I'm looking from the writers' perspective, and from the audience, we're tackling. I read a lot of docs and never have seen this term. I'm referring to FB, Google, Amazon. I have my Alexa skill and couple of FB apps. But maybe for our audience, it's the right term.

Reply:
```suggestion
## Using the Resilience API
```